WORKSPACE

```diff
@@ -13,6 +13,18 @@
 # limitations under the License.
 workspace(name = "bazel_toolchains")
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
```

Comment:
Mention this uses pre-generated configs for the latest available Bazel version and Ubuntu 16.04 toolchain container. Point to the section of the README you copied this from.

Also mention the link is actually unstable (i.e., sha256 digest may change without notice). Also, pre-generated configs are provided on a best-effort basis and don't have any availability guarantees and shouldn't be used in production systems. Point to the README on how users can generate their own configs.

Reply:
Done.